.github/workflows/main-workflow.yml

```diff
@@ -12,15 +12,12 @@ jobs:
 
     - name: tshirts fail
       run: |
-        g++ -Wall -Wextra -Werror tshirts.cpp -o tshirts.out
-        bash .github/workflows/expectfail.sh ./tshirts.out
+        g++ -Wall -Wextra tshirts.cpp -o tshirts.out

```

Comment:
This will only compile, not run. Retain the `./tshirts.out` after the `g++...` line.
Same for the others